src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -264,6 +265,8 @@ public DocumentConventions()
 
             _httpClientType = typeof(HttpClient);
             _createHttpClient = handler => new HttpClient(handler);
+
+            _disposeCertificate = true;
```

Comment:
this is a breaking change, why it was introduced to 5.4?

Reply:
removed